x/fundraising/client/cli/flags.go

```diff
@@ -10,7 +10,7 @@ const (
 	FlagAuctionStatus = "status"
 	FlagAuctionType   = "type"
 	FlagBidderAddr    = "bidder-addr"
-	FlagIsWinner      = "is_winner"
+	FlagIsMatched     = "is_matched"
```

Comment:
This flag was incorrect in the first place. Let's use dash instead of underscore.
```suggestion
	FlagIsMatched     = "is-matched"
```

Reply:
I changed as suggested.